sbt-play-ebean/src/main/scala/play/ebean/sbt/PlayEbean.scala

```diff
@@ -25,7 +40,24 @@ object PlayEbean extends AutoPlugin {
 
   override def trigger = noTrigger
 
-  override def projectSettings = inConfig(Compile)(scopedSettings) ++ unscopedSettings
+  override def projectSettings = {
+    val querySettings = Seq(
+      playEbeanQueryGenerate := false,
+      playEbeanQueryEnhance := false,
+      playEbeanQueryDestDirectory := "app",
```

Comment:
The query beans I see don't seem to have been hand modified after generation, but there's a lot of them, so I admittedly could have missed something: https://github.com/ebean-orm/avaje-ebeanorm-examples/tree/master/a-basic/src/main/java/org/example/domain/query

Can you let me know which one was modified and how?